expenses/config.py

```diff
@@ -1,5 +1,30 @@
 from google.cloud import storage
 import os
+from slack import WebClient
+from slack.errors import SlackApiError
 
 storage_client = storage.Client()
 bucket = storage_client.bucket(os.environ["GOOGLE_STORAGE_BUCKET"])
+
+slack_client = WebClient(token=os.environ.get('SLACK_API_TOKEN', ''))
+
+
+def send_slack_notification(requisition):
+    # Checks if slack token is provide as env var
+    if slack_client.token and requisition.created_by.slack_id:
+        if requisition.status == "Submitted":
```

Comment:
I don't love the direct string comparison in the event the status name ever changes, but it's not a huge deal

Reply:
I changed it to use the status enum, thanks for pointing this out